themes/powerline/powerline.base.bash

```diff
@@ -41,6 +41,24 @@ function __powerline_user_info_prompt {
   [[ -n "${user_info}" ]] && echo "${user_info}|${color}"
 }
 
+function __powerline_terraform_prompt() {
+  local terraform_workspace=""
+
+  if [ -d .terraform ]; then
+    terraform_workspace="$(command terraform workspace show 2>/dev/null)"
+    [[ -n "${terraform_workspace}" ]] && echo "${TERRAFORM_CHAR}${terraform_workspace}|${TERRAFORM_THEME_PROMPT_COLOR}"
+  fi
+}
+
+function __powerline_terraform_prompt() {
+  local terraform_workspace=""
+
+  if [ -d .terraform ]; then
+    terraform_workspace="$(terraform_workspace_prompt)"
+    [[ -n "${terraform_workspace}" ]] && echo "${TERRAFORM_CHAR}${terraform_workspace}|${TERRAFORM_THEME_PROMPT_COLOR}"
+  fi
+}
+
```

Comment:
* I didn't notice the `__powerline_terraform_prompt()` at first, but the general pattern for functions here is to leave out the `()`.

* Technically you don't need the `[ -d .terraform ]` check, but shouldn't hurt to leave it.

Reply:
sublime automatically adds this for me :-)
I have removed it

and I think it is good practice to leave the check